samza-sql/src/main/java/org/apache/samza/sql/util/SamzaSqlQueryParser.java

```diff
@@ -101,6 +101,8 @@ public String getSql() {
   public static QueryInfo parseQuery(String sql) {
     Planner planner = createPlanner();
     SqlNode sqlNode;
+    // Remove trailing semi-colons
+    sql = sql.replaceAll(";+$", "");
```

Comment:
Would be great if we can extract the regex to match the semi-colon into a appropriately named constant to improve readability.